modules/date.js

```diff
@@ -0,0 +1,8 @@
+import DateTime from '../node_modules/luxon/src/datetime.js';
+import { time } from '../index.js'; /* eslint-disable-line */
+
+const timeNow = () => {
+  time.innerHTML = `${DateTime.now().toLocaleString(DateTime.DATE_MED)}`;
+};
```

Comment:
- [ ] Please display the current time as well as required 

![image](https://user-images.githubusercontent.com/75126481/165149727-54e815b6-64db-47fd-adc6-b0ccab13d29b.png)